configure.ac

```diff
@@ -439,6 +439,11 @@ AC_LINK_IFELSE([AC_LANG_PROGRAM([[
 
 AC_SUBST(RKT_DEFINES_FOR_ENTER)
 
+# Check whether we can build TPM support code
+AC_CHECK_HEADER(trousers/tss.h, [TPM_TAGS=tpm], [], [AC_INCLUDES_DEFAULT])
```

Comment:
I don't have strong feelings - I think there's maybe a stronger argument for a disable flag than an enable one?